packages/@aws-cdk/aws-amplify/package.json

```diff
@@ -73,13 +73,23 @@
     "pkglint": "^0.39.0"
   },
   "dependencies": {
-    "@aws-cdk/core": "^0.39.0"
-  },
-  "peerDependencies": {
-    "@aws-cdk/core": "^0.39.0"
+    "@aws-cdk/core": "^0.39.0",
+    "@aws-cdk/aws-iam": "^0.39.0",
+    "@aws-cdk/aws-certificatemanager": "^0.39.0"
   },
   "engines": {
     "node": ">= 8.10.0"
   },
-  "stability": "experimental"
-}
\ No newline at end of file
+  "stability": "experimental",
+  "peerDependencies": {
+    "@aws-cdk/aws-iam": "^0.39.0",
+    "@aws-cdk/aws-certificatemanager": "^0.39.0",
+    "@aws-cdk/core": "^0.39.0"
+  },
+  "awslint": {
+    "exclude": [
+      "from-method:@aws-cdk/aws-amplify.Branch",
```

Comment:
Why?

Reply:
Because creating from methods for these resources has literally no value to anyone. There's nothing you can do with these resources once they're imported.

Reply:
We should have a way to provide an explanation in the "exclude"...